phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/ui/SearchBox.java

```diff
@@ -1,31 +1,34 @@
 package org.phoenicis.javafx.views.mainwindow.ui;
 
-import javafx.event.ActionEvent;
-import javafx.event.EventHandler;
-import org.phoenicis.javafx.views.common.ThemeManager;
-
 import javafx.beans.property.StringProperty;
-import javafx.scene.control.TextField;
 import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
 import javafx.scene.layout.AnchorPane;
 
 import java.util.function.Consumer;
 
 public class SearchBox extends AnchorPane {
     private TextField searchField;
     private Button clearButton;
+    private Consumer<String> onSearch;
```

Comment:
In addition if you want to throw an error if `onSearch` is null, we don't require a new PR, because this is the current behavior, right?